applications/machine_learning/README.rst

```diff
@@ -284,11 +309,14 @@ The following configuration files can be defined for any supported board:
 Multi-image builds
 ------------------
 
-The Thingy:53 uses multi-image build with the following child images:
+The Thingy:53 and nRF53 Development Kit use multi-image build with the following child images:
 
 * MCUboot bootloader
 * Bluetooth HCI RPMsg
 
+Application-specific configuration for the mentioned child images can be defined in the board-specific directory in the :file:`applications/machine_learning/configuration/` directory.
```

Comment:
Is this something that must be done? If that is the case, it's better like this:
```suggestion
Define the application-specific configuration for the mentioned child images in the board-specific directory in the :file:`applications/machine_learning/configuration/` directory.
```

Reply:
For MCUboot configuration it's optional. If it's not defined, default configuration is used

Reply:
I will use `You can define ...`